style.css

```diff
@@ -34,14 +49,102 @@ main {
   width: 100%;
 }
 
+ul {
+  list-style-type: none;
+}
+
+#headline ul {
+  list-style-type: none;
+}
+
 #toolbar nav ul {
   display: flex;
   justify-content: space-between;
   list-style-type: none;
 }
 
-#toolbar nav ul li {
+.details .details-links a {
+  width: 45%;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+.details .web-languages li,
+.card-works .card .card-description .web-languages li,
+.card-works .card .card-description-1 .web-languages li,
+.card-works .card .card-description-2 .web-languages li,
+.card-works .card .card-description-3 .web-languages li {
+  display: grid;
+  grid-template-rows: repeat(1, 1fr);
+  background-color: var(--color11);
+  padding: 4px 12px;
+  border-radius: 8px;
+  margin: 10px 0;
+}
+
+#headline ul li {
+  display: inline-flex;
+  padding: 0 4px;
+  margin: 10px 0;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 12px;
+  font-size: 16px;
+}
+
+/* #toolbar nav ul li {
   display: inline;
+} */
```

Comment:
- [ ] It's a good thing you currently load this part of your portfolio dynamically. How about removing the whole of this completely? Commenting old pieces of code in a workspace makes the code looks kinda messy :hammer_and_wrench: